e2e-tests/tests.rs

```diff
@@ -136,9 +137,22 @@ async fn test_orchestrator_workflow(#[case] l2_block_number: String) {
     println!("Loading .env file");
     dotenvy::from_filename_override(".env.test").expect("Failed to load the .env file");
 
-    let queue_params = QueueArgs {
-        prefix: get_env_var_or_panic("MADARA_ORCHESTRATOR_AWS_PREFIX"),
-        suffix: get_env_var_or_panic("MADARA_ORCHESTRATOR_SQS_SUFFIX"),
+    // E2E tests assume that we are not passing ARNS
+    let aws_prefix = get_env_var_optional_or_panic("MADARA_ORCHESTRATOR_AWS_PREFIX");
+
+    let queue_params = match aws_prefix {
+        Some(prefix) => QueueArgs {
+            queue_template_identifier: orchestrator::types::params::AWSResourceIdentifier::Name(format!(
+                "{}_{}",
+                prefix,
+                get_env_var_or_panic("MADARA_ORCHESTRATOR_AWS_SQS_QUEUE_IDENTIFIER",)
+            )),
+        },
+        None => QueueArgs {
+            queue_template_identifier: orchestrator::types::params::AWSResourceIdentifier::Name(get_env_var_or_panic(
+                "MADARA_ORCHESTRATOR_AWS_SQS_QUEUE_IDENTIFIER",
+            )),
+        },
     };
```

Comment:
> [!NOTE]
> _**Nitpick**_

As far as refactoring goes, something like this would probably be cleaner:
```suggestion
    // E2E tests assume that we are not passing ARNS
    let aws_prefix = get_env_var_optional_or_panic("MADARA_ORCHESTRATOR_AWS_PREFIX");
    let aws_identifier =  get_env_var_or_panic("MADARA_ORCHESTRATOR_AWS_SQS_QUEUE_IDENTIFIER");

    let queue_params = match aws_prefix {
        Some(prefix) => QueueArgs {
            queue_template_identifier: orchestrator::types::params::AWSResourceIdentifier::Name(format!(
                "{}_{}",
                prefix,
                aws_identifier,
            )),
        },
        None => QueueArgs {
            queue_template_identifier: orchestrator::types::params::AWSResourceIdentifier::Name(aws_identifier),
        },
    };
```

Reply:
Oh, alright, seems valid!